src/Microsoft.Framework.Caching.SqlServer/SqlServerCacheOptions.cs

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    public class SqlServerCacheOptions
+    {
+        private readonly TimeSpan MinimumExpiredItemsDeletionInterval;
+        private TimeSpan _expiredItemsDeletionInterval;
+
+        public SqlServerCacheOptions()
+        {
+            _expiredItemsDeletionInterval = MinimumExpiredItemsDeletionInterval;
```

Comment:
@Tratcher : update the PR. I removed the checks from here as we actually them in SqlServerCache.cs itself.